reports/variant_report_latest.md

```diff
@@ -24,6 +24,7 @@ BA.2.75 does appear to circulate globally, albeit at such low share that it is n
 Countries with clusters of BA.2.75 include: UK, US, New Zealand, Japan, Indonesia, Germany, Israel.
 Known countries of importation are: Nepal/India (for some Japanese cases), France (for some Israeli)
```

Comment:
Not sure I parse what this means. Does this mean imported from Nepal from Japan, or vice versa?
Maybe changing to say "Countries where the source of importation is known are Japan (from India/Nepal) and Israel (from France)." (Or change the order if I got it wrong?)